Documentation/Input/Pointers.md

```diff
@@ -17,12 +17,12 @@ Focus Enter / Exit | Raised on the game object gaining focus when the first poin
 Pointer Down / Dragged / Up / Clicked | Raised to report pointer press, drag and release. | [`IMixedRealityPointerHandler`](xref:Microsoft.MixedReality.Toolkit.Input.IMixedRealityPointerHandler)
 Touch Started / Updated / Completed | Raised by touch-aware pointers like [**PokePointer**](xref:Microsoft.MixedReality.Toolkit.Input.PokePointer) to report touch activity. | [`IMixedRealityTouchHandler`](xref:Microsoft.MixedReality.Toolkit.Input.IMixedRealityTouchHandler)
 
-To handle them you must implement the corresponding interface in your script. Aditionally, if you want your game object to receive the events raised on all game objects instead of just the ones raised on itself and its children, you must register as a global event listener calling [`Register`](xref:Microsoft.MixedReality.Toolkit.IMixedRealityEventSystem.Register*) on the input system.
+To handle them you must implement the corresponding interface in your script. Aditionally, if you want your game object to receive the events raised on all game objects instead of just the ones raised on itself and its children, you must register as a global event listener calling [`RegisterHandler`](xref:Microsoft.MixedReality.Toolkit.IMixedRealityEventSystem.RegisterHandler*) on the input system with the interface(s) you want to receive events for.
 
 > [!WARNING]
```

Comment:
We could downgrade this to just a note saying that those types of events should not be marked as used by global handlers.